### PR TITLE
Abstract access to injected globals

### PR DESCRIPTION
**Pull request by mszabo-wikia**
## Which problem is this PR solving?
- Split from https://github.com/jaegertracing/jaeger-ui/pull/1212

## Short description of the changes
Create a wrapper module for accessing globals injected by the build system. This makes it easier to rename or adjust those later.

### CODE REVIEW

File: packages/jaeger-ui/src/utils/constants.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2023 Uber Technologies, Inc.
```
Comment (yurishkuro) [inline packages/jaeger-ui/src/utils/constants.tsx:1]:
```suggestion
// Copyright (c) 2023 The Jaeger Authors.
```

File: packages/jaeger-ui/src/utils/version/get-version.test.js [hunk 2]
```diff
@@ -34,9 +34,9 @@ describe('getVersion()', () => {
     console.warn = oldWarn;
   });
 
-  describe('`window.getVersion` is not a function', () => {
+  describe('`window.getJaegerVersion` is not a function', () => {
```
Comment (yurishkuro) [inline packages/jaeger-ui/src/utils/version/get-version.test.js:37]: what is the origin of this change? I only see the test change, not the main code.

Comment (mszabo-wikia) [reply packages/jaeger-ui/src/utils/version/get-version.test.js:37]: It looks like this test was testing for the wrong function name - it should be `window.getJaegerVersion` as per https://github.com/jaegertracing/jaeger-ui/blob/dda3b97edccac392324ddfc4d90487377c1ca47e/packages/jaeger-ui/src/utils/version/get-version.tsx.

**Review by yurishkuro — APPROVED** [review]
👍

**Pull request merged**